pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -139,10 +140,11 @@ let
     '';
 
   configureFlags = [
-    "--enable-optimizations"
     "--enable-shared"
     "--with-threads"
     "--enable-unicode=ucs${toString ucsEncoding}"
+  ] ++ optionals enableOptimizations [
+    "--enable-optimizations"
```

Comment:
We might prefer to avoid Linux rebuilds on staging-next by keeping the flag order for now.